assets/data/effects/drauven/drauven_pc_drauven_guerilla_upgrade_movement.json

```diff
@@ -0,0 +1,47 @@
+{
+"id": "drauven_pc_drauven_guerilla_upgrade_movement",
+"info": {
+	"dataVersion": 1,
+	"sourceFile": "drauven/drauven_pc_drauven_guerilla_upgrade_movement",
+	"author": "Bipen",
+	"tags": [ "okToMarkDone", "sneaky" ]
```

Comment:
To make this effect work when there's no action points left, I think we just need to remove `okToMarkDone`

Reply:
thanks for pointing it out it works properly now